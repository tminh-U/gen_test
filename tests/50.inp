7
19 6 3 14 7 18 12 
8
5 5
1 4
4 7
5 6
7 7
1 7
2 7
4 7
