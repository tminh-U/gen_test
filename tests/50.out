2
7
8
4
2
25
24
8
